terraform/deployments/govuk-publishing-platform/dns.tf

```diff
@@ -31,6 +31,8 @@ resource "aws_route53_zone" "internal_private" {
 resource "aws_acm_certificate" "workspace_public" {
   domain_name = "*.${local.workspace_external_domain}"
 
+  subject_alternative_names = local.workspace == "ecs" ? ["*.${local.workspace}.${var.publishing_service_domain}"] : null
```

Comment:
I wonder if `terraform.workspace == "default"` would be clearer here. Possibly less likely to break if we decide to change the default local.workspace name too.

Reply:
I want the terraform workspace to exist in `default.tf` only, it is confusing to see terraform.workspace/workspace side by side in other places.

Reply:
What do you think about adding a `local.is_default_workspace` variable in that file then?

Reply:
happy to add that to defaults.tf